mintapi/signIn.py

```diff
@@ -336,6 +336,7 @@ def sign_in(
     driver.get(url)
     if not beta:
         element = driver.find_element_by_link_text("Sign in")
+        time.sleep(1)
```

Comment:
Instead of sleeping, you should use WebDriverWait for the element in question. That way there isn't a 1 second sleep unless it's required.

Reply:
I do agree that its better to avoid using `time.sleep` when possible. However, when trying to using WebDriverWait, the error still occurs. So its odd that its finding the sign in button but the click is not working without a delay. Its also a bit difficult to debug since a linebreak before the click gives it enough time to work as expected.

```
        element = WebDriverWait(driver, 20).until(
            expected_conditions.presence_of_element_located(
                (By.LINK_TEXT, "Sign in")
            )
        )
```

Reply:
@CalvinChanCan can you commit the version with WebDriverWait that doesn't solve it? Just want to take a look and see if anything sticks out as to why.

Reply:
Thanks @burkematthew! I made committed the version with WebDriverWait that doesn't resolve.